snapshot/pkg/volume/hostpath/processor.go

```diff
@@ -61,6 +61,9 @@ func (h *hostPathPlugin) SnapshotCreate(pv *v1.PersistentVolume, tags *map[strin
 		return nil, nil, fmt.Errorf("invalid PV spec %v", spec)
 	}
 	path := spec.HostPath.Path
+	if _, err := os.Stat(path); err != nil {
```

Comment:
validate file mode as well? in case path exists but it is a file

Reply:
Good point. Will fix that.